python/arcticc/__init__.py

```diff
@@ -1 +1,9 @@
+# file from ArcticDB
 __path__ = __import__("pkgutil").extend_path(__path__, __name__)
+
+# Use Man's API backwards compatibility layer if it is installed.
+# This will not do anything outside of Man Group systems.
+try:
+    from man.arcticdb.arcticdb_renamer import *
+except ImportError:
+    pass
```

Comment:
I don't like the fact that this copy of the file behaves differently from the others.

There could be strange situations where `man.arcticdb` exists but something it needs doesn't and you only get the exception some of the time.

I wonder if it's possible to omit this file from the external repo and only have it in the internal bridge?

Reply:
I don't know of a way to omit this file, since the `arcticc` namespace package exists in this repo and also internally.

Reply:
When I was playing with namespace packages, I noticed it's possible to have a package without `__init__.py` using the `sys.path` method.

I think it might not conflict with the `pkgutil` method, i.e. you might be able to remove this file.

The question then becomes if the protobuf files are compatible between `arcticc` and `arcticdb`.